prov/util/src/util_attr.c

```diff
@@ -687,13 +719,24 @@ int ofi_check_rx_attr(const struct fi_provider *prov,
 		return -FI_ENODATA;
 	}
 
+	if (!rm_enabled &&
+	    user_attr->total_buffered_recv > prov_attr->total_buffered_recv) {
+		FI_INFO(prov, FI_LOG_CORE,
+			"Total buffered recv size exceeds supported size\n");
+		FI_INFO_CHECK_VAL(prov, prov_attr, user_attr,
+				  total_buffered_recv);
+		return -FI_ENODATA;
+	}
+
 	return 0;
 }
 
 int ofi_check_tx_attr(const struct fi_provider *prov,
-		      const struct fi_tx_attr *prov_attr,
+		      const struct fi_info *prov_info,
```

Comment:
Why was this change needed? prov_info is not getting used in the function.

Reply:
will remove them